### PR TITLE
disable dataset viewer for 'nfaa' tag

### PR DESCRIPTION
**Pull request by severo**
See https://huggingface.slack.com/archives/C042H6N5JR0/p1714771691945989 (internal)

On the first backfill cron job after this PR is merged, all the datasets with the "nfaa" tag will be deleted from the dataset viewer's cache.

### CODE REVIEW

**Review by albertvillanova — APPROVED** [review]
Thanks.

File: libs/libcommon/src/libcommon/constants.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 PROCESSING_STEP_CONFIG_PARQUET_AND_INFO_ROW_GROUP_SIZE_FOR_BINARY_DATASETS = 100
 PARQUET_REVISION = "refs/convert/parquet"
 
+UNSUPPORTED_TAG_NFAA = "not-for-all-audiences"
```
Comment (Pierrci) [inline libs/libcommon/src/libcommon/constants.py:36]: we have other tags in addition to the official one for which we want to apply the same policy if possible, see https://github.com/huggingface/moon-landing/blob/ec6c0f42e48d3479a09b85a1a3dc81915cb88920/server/repo_types/Tags.ts#L2 (I think it's ok to just copy/paste, we'll try to ping you if/when we add more)

**Comment by Pierrci** [comment]
cc @allendorf @No-m4d for vis on this

File: libs/libcommon/src/libcommon/constants.py (hunk 1 above)
Comment (severo) [reply libs/libcommon/src/libcommon/constants.py:36]: OK, done in https://github.com/huggingface/dataset-viewer/pull/2823

**Pull request merged**